src/blas/impl/KokkosBlas3_gemm_impl.hpp

```diff
@@ -340,7 +340,10 @@ void impl_team_gemm_block(const TeamHandle& team, const ViewTypeC& C, const View
 #if defined(__CUDA_ARCH__) || !defined(KOKKOS_ENABLE_OPENMP)
     Kokkos::parallel_for(Kokkos::ThreadVectorRange(team,blockB1/4), [&] (const int B_j) {
 #else
+  #if (KOKKOS_COMPILER_GNU < 485 )
```

Comment:
This probably is not correct for intel. You probably need to nest this, or alternatively add a OMP_SIMD_EXISTS or whatever variable inside of Kokkos_Macros.hpp (there are similar variables for other pragma features, model it based on those). But that would require a change in Kokkos ...

Reply:
@crtrott thanks for catch, just pushed fix. Rerunning with gcc and intel